src/main/webapp/pages/common/banner.jsp

```diff
@@ -92,18 +94,33 @@ function getCsrfToken() {
  		<div style="display: block">
 			<%
 				UserSessionData usd = null;
+				NotificationService ns = new NotificationService(); 
 				if (request.getSession().getAttribute(IActionConstants.USER_SESSION_DATA) != null) {
 					usd = (UserSessionData) request.getSession().getAttribute(IActionConstants.USER_SESSION_DATA);
+					String saveDate = ns.getLoginFromCombinedId(usd.getLoginName()).toString();
+
 			%>
+
+			<script type="text/javascript">
+				function showExpiaryNotificationMessage() {
+					var now = moment().format('DD-MMMM-YYYY, h:mm:ssA');
+				    var date_string = '<%=Encode.forJavaScript(saveDate)%>';
+                    var expiration = moment(date_string).format("YYYY-MM-DD");
+                    var current_date = moment().format("YYYY-MM-DD");
+                    var hours = moment(expiration).diff(current_date, 'hours');
+                       alert(now + ' Your password will expire on '  +  date_string  + 'You have ' + hours + ' hours left .Please set new password '); 
```

Comment:
@gitcliff 
can you instead get the message form the message files for translation ??